docs/server-introduction.md

```diff
@@ -0,0 +1,53 @@
+---
+id: server-introduction
+title: Introduction to the Temporal server
+sidebar_label: Introduction
+---
+
+## Overview
+
+The [Temporal server](https://github.com/temporalio/temporal) is a micro-service orchestration platform. It can be thought of as a layer in your technology stack that exists between your compute servers and your executable source code. It is completely open source and can be run in a variety of cloud or on-premise environments.
+
+The Temporal server tracks the state of Workflow functions and automatically handles timeouts and retries for units of code. This enables developers to focus on writing business logic instead of application reliability infrastructure.
+
+## Install for local development environment
+
+The fastest way to start building Workflow applications and test out the system is to follow the [quick install guide](/docs/server-quick-install), which will get the Temporal server up and running on your local machine.
+
+If you want to see how Temporal runs on a Kubernetes cluster, the fastest way is to use this [Helm chart](https://github.com/temporalio/helm-charts).
+
+## Install for a live environment
+
+To run Temporal in a cloud or on-premise live environment, [include the Temporal server package](/docs/server-options) and run it as a Go application.
```

Comment:
can we include signup link for temporal cloud email list here? just to do the email capture right. https://temporal.us17.list-manage.com/subscribe/post?u=2334a0f23e55fd1840613755d&amp;id=bbbbd4709f

Reply:
One of the next things I was going to do, but separately, was create a "Cloud" tab in the docs with a page that has this messaging and a link to sign up to the email list.

I don't think we should covering up open source server implementation info with a message about our cloud offering though?

Reply:
its your call :) just a suggestion to put one link, i also dont agree with blanketing the docs with ads but this i considered tasteful enough

Reply:
I will revise this to link to this new cloud info page that has this message:
https://github.com/temporalio/documentation/pull/212